.changeset/nine-ducks-wink.md

```diff
@@ -2,4 +2,17 @@
 "@itwin/presentation-components": minor
 ---
 
-Replaced `react-select` with `iTwinUI` comboboxes, limited visible options to 100, and added a non-pickable option prompting users to filter beyond the limit.
+Replaced `react-select` with [iTwinUI's ComboBox](https://itwinui.bentley.com/docs/combobox).
+
+This affects [`PresentationInstanceFilterBuilder`, `NavigationPropertyEditor`] components.
+
+Changes:
+
+- The number of select options is limited to 100. When more items exist, a non-selectable option is displayed at the bottom of the list, prompting users to provide an items filter. Previously, additional pages of select options was loaded when user scrolled to the bottom of the list.
+
+- Old style:
+    ![alt text](image-1.png)
+    New style:
+    ![alt text](image.png)
```

Comment:
- Formatting of this won't look good when the markdown is rendered. I think we could show this as a table (e.g. like [this](https://github.com/iTwin/viewer-components-react/blob/dc644c1d291c175cad84bb04513efc84c54413d2/packages/itwin/tree-widget/README.md?plain=1#L23-L25)).

- Do we know what will happen with the images upon release, when these changesets are merged into the changelog? I don't think we have any code in place to make sure the images are copied-over, or even know where they should land.

Reply:
Removed images altogether